requirements.txt

```diff
@@ -13,3 +13,6 @@ pytest==5.4.2
 pytest-cov==2.10.0
 pytest-azurepipelines
 coverage
+cmake
+dlib
```

Comment:
I added it because without it the build failed.

Reply:
Can you provide a link to the build?

In cgm-ml there is a build for this that works without definining cmake and dlib

Reply:
https://dev.azure.com/cgmorg/ChildGrowthMonitor/_build/results?buildId=1074&view=results

Reply:
Ok this is fine for now, I think it is because the python3 docker image doesn't have cmake installed as a system library.

But it is fine for now to put it in the requirements.txt